### PR TITLE
added myself

### PR DESCRIPTION
**Pull request by pritambera2000**
i added my data

### CODE REVIEW

**Comment by aryasoni98** [comment]
Follow Pattern: 
---		

name: YOUR_NAME		
institution: INSTITUTION_NAME		
image: PROFILE_PICTURE_FILENAME		
quote: YOUR_QUOTE		
github: USERNAME
   
---

File: content/participant/pritambera2000.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (aryasoni98) [inline content/participant/pritambera2000.md:1]: Add Space <Enter>

File: content/participant/pritambera2000.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---		
+name: Pritam Bera		
+institution: VU		
+image: pritam.JPG		
+quote: Knowledge Comes from Experience.		
+github: pritambera2000
```
Comment (aryasoni98) [inline content/participant/pritambera2000.md:6]: Add Space < Enter>

**Review by aryasoni98 — CHANGES_REQUESTED** [review]

**Comment by pritambera2000** [comment]
Please Review

**Review by aryasoni98 — APPROVED** [review]

**Pull request merged**